how-to-release.md

```diff
@@ -84,65 +94,63 @@ for file in ./*.jar ; do shasum -a 1 $file > $file.sha1 ; done
 for file in ./*.jar ; do shasum -a 256 $file > $file.sha256 ; done
 ```
 
-*Note*: In the future, we might want to automate the generation of checksums via gradle.
+>**Note**: Generation of checksums via gradle is planned to be automated in the future releases.

```

Comment:
either we remove this line or replace it with the original statement.

> *Note*: In the future, we might want to automate the generation of checksums via gradle.

Reply:
Replace with old one